src/pcapi/core/educational/factories.py

```diff
@@ -18,15 +21,32 @@ class EducationalYearFactory(BaseFactory):
     class Meta:
         model = models.EducationalYear
 
-    adageId = factory.Sequence(lambda number: str(6 + number))
+    adageId = factory.Sequence(lambda number: str(_get_current_educational_year_adage_id() + number))
     beginningDate = factory.Sequence(
-        lambda number: datetime.datetime(2020, 9, 1) + datetime.timedelta(days=365 * number)
+        lambda number: datetime.datetime(_get_current_educational_year(), 9, 1) + datetime.timedelta(days=365 * number)
```

Comment:
Petite correction de mon code, ici on peut utiliser relativedelta pour éviter de se tromper avec les années bissextiles 😬 
```suggestion
        lambda number: datetime.datetime(_get_current_educational_year(), 9, 1) + relativedelta(years=number)
```